CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### Added
+- Add `'type' : 'button'` to buttons
+
```

Comment:
We usually omit the redundant `Add` keyword in the `Added` section. (And the `develop` header is missing)

```suggestion

## develop

### Added
- `'type' : 'button'` to buttons to prevent submitting forms

```